common/_static/custom.css

```diff
@@ -1,27 +1,94 @@
+@font-face {
```

Comment:
I believe it's what we do on http://get.asp.net to ensure fonts work well on OSX